packages/synapse-react-client/src/components/FullWidthAlert/FullWidthAlert.tsx

```diff
@@ -24,8 +25,11 @@ export type AlertButtonConfig = {
   | { href?: string }
 )
 
-export interface FullWidthAlertProps extends AlertProps {
-  variant: string
+export type FullWidthAlertVariant = 'warning' | 'info' | 'danger' | 'success'
+
+export interface FullWidthAlertProps {
+  variant?: FullWidthAlertVariant
+  show?: boolean
```

Comment:
`show` was previously added as a prop by extending bootstrap AlertProps. Now, explicitly specify the props to maintain the same interface, other than removing removing the `transition` prop, since it didn't seem to be used